core/common/src/annotation/TextBlock.ts

```diff
@@ -411,6 +411,20 @@ export class Paragraph extends TextBlockComponent {
  */
 export type TextBlockJustification = "left" | "center" | "right";
 
+/** Describes the margins around the content inside a [[TextBlock]]. It's measured in meters.
+ * @beta
+ */
+export interface TextBlockMargins {
+  /** The left margin. Must be a positive integer >= 0. Negative values are disregarded */
```

Comment:
It's measured in meters, it doesn't need to be an integer.

Reply:
Should be fixed now